Mod/Classes/New/PlayerGhostInventory.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Mod;
+using TowerFall;
+
+namespace New
```

Comment:
Any reason you use a different namespace? I used Mod for all my patched and new classes.